core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -40,6 +40,7 @@ oppia.directive('audioBar', [
             EVENT_AUTOPLAY_AUDIO, BrowserCheckerService) {
           $scope.audioBarIsExpanded = false;
           $scope.hasPressedPlayButtonAtLeastOnce = false;
+          $scope.progressBarIsEnabled = true;
```

Comment:
(Not sure about the UI behavior) are we enabling/disabling the progress bar or it show/hide? If the alter one then change this variable to `showProgressBar`.

Reply:
sir we were enabling/disabling the progress bar.
``` progressBarIsEnabled ``` is used with ``` ng-if  ```

Reply:
I think the expected behavior suggested in #6214 is to hide the progress bar for "auto-gen" audio.

Reply:
yes sir